cli.js

```diff
@@ -12,10 +12,12 @@ const start = require('./start')
 const generate = require('./generate')
 const generateReadme = require('./generate-readme')
 const printRoutes = require('./print-routes')
+const renderDocs = require('./docs')
 
 commist.register('start', start.cli)
 commist.register('generate', generate.cli)
 commist.register('readme', generateReadme.cli)
+commist.register('docs', renderDocs)
```

Comment:
We'll need to disable this in Node < 10 I think

Reply:
Fair, I'll add a check in the render docs fn

Reply:
you cannot require that file on node 6 and possibly 8. We might just bump the major anyway, would you be ok with that? (This might land on a next branch then).

Reply:
I'm happy with that 😊

Reply:
I've got around this by removing async/await, and just using the sync fs funcs, sync should be fine for this use-case I think